zerver/models.py

```diff
@@ -1495,9 +1495,9 @@ class UserBaseSettings(models.Model):
     TWITTER_EMOJISET = "twitter"
     EMOJISET_CHOICES = (
         (GOOGLE_EMOJISET, "Google modern"),
-        (GOOGLE_BLOB_EMOJISET, "Google classic"),
         (TWITTER_EMOJISET, "Twitter"),
         (TEXT_EMOJISET, "Plain text"),
+        (GOOGLE_BLOB_EMOJISET, f"Google classic ({_('deprecated')})"),
```

Comment:
Maybe it's better to add the "deprecated" text directly in the settings menu like the tooltip, and that would also make it easier to make it a link.